CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 All notable changes for each version of this project will be documented in this file.
 
 ## 6.2.0
+- `igx-input-group`
+    - Display density of the `igx-input-group` component can be set using the injection token `DisplayDensityToken` and providing a value (comfortable, cosy or compact) on an application or a component level.
```

Comment:
For `igx-input-group` the display density could also be set per instance via the provided input property.
<igx-input-group [displayDensity]="'cosy'"></igx-input-group>